### PR TITLE
Add documentation for telegram_bot reply_to_message_id option

### PR DESCRIPTION
**Pull request by TheKalpit**
## Proposed change
Adds documentation change for `telegram_bot` integration update.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103566
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/telegram_bot.markdown [hunk 1]
```diff
@@ -60,6 +60,7 @@ Send a photo.
...
 | `inline_keyboard`      | yes      | List of rows of commands, comma-separated, to make a custom inline keyboard with buttons with associated callback data or external URL (https-only). Example: `["/button1, /button2", "/button3"]` or `[[["Text btn1", "/button1"], ["Text btn2", "/button2"]], [["Google link", "https://google.com"]]]` |
 | `message_tag`          | yes      | Tag for sent message. In `telegram_sent` event data: {% raw %}`{{trigger.event.data.message_tag}}`{% endraw %}                                                                                                                                                                                            |
+| `reply_to_message_id`      | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                           |
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`  | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                       |
```

File: source/_integrations/telegram_bot.markdown [hunk 2]
```diff
@@ -82,6 +83,7 @@ Send a video.
...
 | `keyboard`             | yes      | List of rows of commands, comma-separated, to make a custom keyboard. `[]` to reset to no custom keyboard. Example: `["/command1, /command2", "/command3"]`                                                                                                                                               |
 | `inline_keyboard`      | yes      | List of rows of commands, comma-separated, to make a custom inline keyboard with buttons with associated callback data or external URL (https-only). Example: `["/button1, /button2", "/button3"]` or `[[["Text btn1", "/button1"], ["Text btn2", "/button2"]], [["Google link", "https://google.com"]]]` |
+| `reply_to_message_id`      | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                           |
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`  | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                       |
```

File: source/_integrations/telegram_bot.markdown [hunk 3]
```diff
@@ -105,6 +107,7 @@ Send an animation.
...
 | `inline_keyboard`      | yes      | List of rows of commands, comma-separated, to make a custom inline keyboard with buttons with associated callback data or external URL (https-only). Example: `["/button1, /button2", "/button3"]` or `[[["Text btn1", "/button1"], ["Text btn2", "/button2"]], [["Google link", "https://google.com"]]]` |
 | `message_tag`          | yes      | Tag for sent message. In `telegram_sent` event data: {% raw %}`{{trigger.event.data.message_tag}}`{% endraw %}                                                                                                                                                                                            |
+| `reply_to_message_id`      | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                           |
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`  | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                       |
```

File: source/_integrations/telegram_bot.markdown [hunk 4]
```diff
@@ -127,6 +130,7 @@ Send a voice message.
...
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`  | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                       |
```

File: source/_integrations/telegram_bot.markdown [hunk 5]
```diff
@@ -149,6 +153,7 @@ Send a sticker.
...
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`  | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                       |
```

File: source/_integrations/telegram_bot.markdown [hunk 6]
```diff
@@ -172,6 +177,7 @@ Send a document.
...
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`  | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                       |
```

File: source/_integrations/telegram_bot.markdown [hunk 7]
```diff
@@ -188,6 +194,7 @@ Send a location.
...
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`  | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                       |
```

File: source/_integrations/telegram_bot.markdown [hunk 8]
```diff
@@ -203,6 +210,7 @@ Send a poll.
 | `open_period`             | yes      | Amount of time in seconds the poll will be active after creation, 5-600.                                                                                                       |
 | `disable_notification`    | yes      | True/false for send the message silently. iOS users and web users will not receive a notification, Android users will receive a notification with no sound. Defaults to False. |
 | `timeout`                 | yes      | Timeout for sending voice in seconds. Will help with timeout errors (poor internet connection, etc)                                                                            |
+| `reply_to_message_id`      | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %}                                                                                                                           |
```
Comment (FletcherAU) [inline source/_integrations/telegram_bot.markdown]:
```suggestion
| `reply_to_message_id`     | yes      | Mark the message as a reply to a previous message. In `telegram_callback` handling, for example, you can use {% raw %}`{{ trigger.event.data.message.message_id }}`{% endraw %} |
```
Fixing the table alignment on this one is out of scope for a review according to Github. Main thing is the padding for the last column on other rows needs to move out by two spaces :)

**Review by FletcherAU — CHANGES_REQUESTED** [review]
Looks great, just need to fix the table formatting to make it easier for the next editor :)

**Comment by TheKalpit** [comment]
@FletcherAU Done. Thanks!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @TheKalpit and @FletcherAU :+1:
